lib/osf-components/addon/components/search-page/template.hbs

```diff
@@ -180,3 +197,121 @@ as |layout|>
         {{/unless}}
     </layout.main>
 </OsfLayout>
+{{#if this.showTooltip1}}
```

Comment:
I had to wrap these in `{{#if}}`s here because otherwise the popover wouldn't render in the right place when the component's state would change (e.g. going from mobile to desktop, vice versa, and if filterable properties were visible or not)